packages/helix-shared-config/src/SitemapHandler.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+const fetchAPI = require('@adobe/helix-fetch');
+const { NamedMapHandler } = require('./NamedMapHandler.js');
+
+const { fetch } = process.env.HELIX_FETCH_FORCE_HTTP1
+  ? fetchAPI.context({ alpnProtocols: [fetchAPI.ALPN_HTTP1_1] })
+  /* c8 ignore next */
+  : fetchAPI;
+
+const wrap = (sitemap) => {
+  if (typeof sitemap === 'object') {
+    return {
+      ...sitemap,
+      reset: () => {
+        fetchAPI.reset();
```

Comment:
doesn't look very _thread safe_, since the `context`  is global.
you should create a `context` for each sitemap.